src/java/com/wolfssl/WolfSSLSession.java

```diff
@@ -2342,6 +2343,24 @@ public long setOptions(long op)
         return setOptions(getSessionPtr(), op);
     }
 
+
+    /**
+     * Gets the options to use for the WOLFSSL structure.
+     * Example options are WolfSSL.SSL_OP_NO_SSLv3
+     *
+     *
+     * @param op      bit mask of options to set
```

Comment:
This @param notation should be removed.  getOptions() does not accept any parameters.  This causes a Javadoc error on OSX:

```
  [javadoc] /Users/chrisc/Desktop/kojo/wolfssljni/src/java/com/wolfssl/WolfSSLSession.java:2352: error: @param name not found
  [javadoc]      * @param op      bit mask of options to set
```